fineract-provider/src/main/java/org/apache/fineract/infrastructure/bulkimport/service/BulkImportWorkbookServiceImpl.java

```diff
@@ -99,8 +99,12 @@ public Long importWorkbook(String entity, InputStream inputStream, FormDataConte
                 final String fileExtension = Files.getFileExtension(fileDetail.getFileName()).toLowerCase();
                 ImportFormatType format = ImportFormatType.of(fileExtension);
                 if (!fileType.contains("msoffice")) {
-                    throw new GeneralPlatformDomainRuleException("error.msg.invalid.file.extension",
-                            "Uploaded file extension is not recognized.");
+                    // We had a problem where we tried to upload the downloaded file from the import options, it was somehow changed the
+                    // extension we use this fix.
+                    if (!fileType.contains("application/vnd.ms-excel")) {
```

Comment:
the double `if` is a little bit weird; it would be nicer if this was a single if, with an `&&` ?